moirae/estimators/online/distributions.py

```diff
@@ -1,11 +1,32 @@
-""" Base online estimators, which are all model-agnostic """
+"""Classes defining different multivariate probability distributions"""
+from abc import abstractmethod
 from warnings import warn
 from typing_extensions import Self
 
 import numpy as np
-from pydantic import Field, field_validator, computed_field, model_validator
+from pydantic import Field, field_validator, computed_field, model_validator, BaseModel
 
-from moirae.estimators.online import MultivariateRandomDistribution
+
+class MultivariateRandomDistribution(BaseModel, arbitrary_types_allowed=True):
+    """
+    Base class to help represent a multivariate random variable
+    """
+
+    @abstractmethod
+    def get_mean(self) -> np.ndarray:
+        """
+        Provides mean (first moment) of distribution
+        """
+        pass
+
+
+class PointEstimate(MultivariateRandomDistribution):
```

Comment:
An "estimate" and a "distribution" are different things. I think the name here should be something like `MultivariateDeltaDistribution` for consistency.

Reply:
Good call